recipes-extended/sysklogd/sysklogd_%.bbappend

```diff
@@ -0,0 +1,7 @@
+inherit systemd
+
+SYSTEMD_SERVICE_${PN}_remove = "syslogd.service"
```

Comment:
could we rather remove sysklogd as well rather than just inhibiting its startup?